src/api/status/public/js/build-log/build-header.js

```diff
@@ -0,0 +1,39 @@
+const buildHeaderTitle = document.getElementById('build-header-title');
+const buildHeaderInfo = document.getElementById('build-header-info');
+const buildSender = document.getElementById('build-sender');
+const buildSenderName = document.getElementById('build-sender-name');
+const buildSenderImg = document.getElementById('build-sender-img');
+const buildGitSHA = document.getElementById('build-git-sha');
+const buildResult = document.getElementById('build-result');
+const buildStarted = document.getElementById('build-started');
+const buildDuration = document.getElementById('build-duration');
+
+function renderBuildInfo({ githubData, startedAt, stoppedAt, result }) {
+  buildHeaderTitle.innerHTML = '';
+  buildHeaderTitle.innerText = 'Build Status';
+  buildSender.setAttribute('href', githubData.sender.html_url);
+  buildSenderName.innerText = githubData.sender.login;
+  buildSenderImg.src = githubData.sender.avatar_url;
+  buildGitSHA.setAttribute('href', githubData.compare);
+  buildGitSHA.innerText = githubData.after.substring(0, 7);
+  buildResult.innerText = result === 0 ? 'Good' : 'Error';
```

Comment:
According to the data ,  ``` result: data.code =1```
![1639114318(1)](https://user-images.githubusercontent.com/55161917/145522510-98a79060-ab6d-46a9-baf4-05d25ef8a9cb.png)
I think the line 19 should changed to  ``` buildResult.innerText = result != 0 ? 'Good' : 'Error'; ```